### ISSUE TITLE
Download JSON option not working on Safari on OS X

### ISSUE THREAD

**Issue by asieira** [swagger-api/swagger-editor#636]
When I run swagger-editor on my OS X machine, it works perfectly on Firefox. But if I open it in Safari, clicking on `Download JSON` simply reloads the YAML editing page I was already on.

This is what the browser console logs show me, maybe this helps:
```
[Log] Safari Injected running (safariInjected.js, line 10)
[Warning] Unexpected CSS token: : (b38ef087.main.css, line 20, x3)
[Warning] Unexpected CSS token: : (b38ef087.main.css, line 23, x5)
[Log] Warn: could not find module util (81b1cc3b.vendor.js, line 50)
[Log] injected.js running now. (injected.js, line 32)
[Log] Dequeuing 0 messages (safariInjected.js, line 36)
[Warning] Automatically scrolling cursor into view after selection change this will be disabled in the next version set editor.$blockScrolling = Infinity to disable this message (81b1cc3b.vendor.js, line 21)
[Warning] Automatically scrolling cursor into view after selection change this will be disabled in the next version set editor.$blockScrolling = Infinity to disable this message (81b1cc3b.vendor.js, line 21)
[Error] TypeError: undefined is not an object (evaluating 'this.currentTask.cb')
	onerror (81b1cc3b.vendor.js, line 65)
	(anonymous function) ([native code], line 0)
[Error] SyntaxError: Unexpected end of script (blob:http://localhost:8080/30afbe05-4766-4a04-9ecc-d0a1eac9090a, line 11)
[Error] Error: undefined is not a function (evaluating 'text.replace(/\t/g,"  ")')
unindent@http://localhost:8080/scripts/81b1cc3b.vendor.js:19:32218
set@http://localhost:8080/scripts/81b1cc3b.vendor.js:19:32536
link@http://localhost:8080/scripts/81b1cc3b.vendor.js:19:20974
http://localhost:8080/scripts/81b1cc3b.vendor.js:9:7201
invokeLinkFn@http://localhost:8080/scripts/81b1cc3b.vendor.js:9:7320
nodeLinkFn@http://localhost:8080/scripts/81b1cc3b.vendor.js:8:60089
compositeLinkFn@http://localhost:8080/scripts/81b1cc3b.vendor.js:8:27153
compositeLinkFn@http://localhost:8080/scripts/81b1cc3b.vendor.js:8:27262
compositeLinkFn@http://localhost:8080/scripts/81b1cc3b.vendor.js:8:27262
publicLinkFn@http://localhost:8080/scripts/81b1cc3b.vendor.js:8:36421
boundTranscludeFn@http://localhost:8080/scripts/81b1cc3b.vendor.js:8:27635
controllersBoundTransclude@http://localhost:8080/scripts/81b1cc3b.vendor.js:8:60655
ngRepeatAction@http://localhost:8080/scripts/81b1cc3b.vendor.js:14:66871
$watchCollectionAction@http://localhost:8080/scripts/81b1cc3b.vendor.js:11:113739
$digest@http://localhost:8080/scripts/81b1cc3b.vendor.js:12:423
http://localhost:8080/scripts/81b1cc3b.vendor.js:12:2421
completeOutstandingRequest@http://localhost:8080/scripts/81b1cc3b.vendor.js:7:60329
http://localhost:8080/scripts/81b1cc3b.vendor.js:8:1120 <p marked="tag.description" class="ng-isolate-scope">
	(anonymous function) (81b1cc3b.vendor.js, line 10)
	(anonymous function) (81b1cc3b.vendor.js, line 9)
	invokeLinkFn (81b1cc3b.vendor.js, line 9)
	nodeLinkFn (81b1cc3b.vendor.js, line 8)
	compositeLinkFn (81b1cc3b.vendor.js, line 8)
	compositeLinkFn (81b1cc3b.vendor.js, line 8)
	compositeLinkFn (81b1cc3b.vendor.js, line 8)
	publicLinkFn (81b1cc3b.vendor.js, line 8)
	boundTranscludeFn (81b1cc3b.vendor.js, line 8)
	controllersBoundTransclude (81b1cc3b.vendor.js, line 8)
	ngRepeatAction (81b1cc3b.vendor.js, line 14)
	$watchCollectionAction (81b1cc3b.vendor.js, line 11)
	$digest (81b1cc3b.vendor.js, line 12)
	(anonymous function) (81b1cc3b.vendor.js, line 12)
	completeOutstandingRequest (81b1cc3b.vendor.js, line 7)
	(anonymous function) (81b1cc3b.vendor.js, line 8)
```

**Comment by mohsen1** [comment]
```
SyntaxError: Unexpected end of script (blob:http://localhost:8080/30afbe0...
```

Safari can't handle large blobs as source of script for web workers. It's a bug in Safari which they fixed in WebKit version 602.1.2+.
Current Safari in OSX has WebKit version	600.8.9. Either use a newer WebKit engine or Chrome

**Comment by asieira** [comment]
Ok, thank you very much for the quick response.
